modules/op-creating-webhooks.adoc

```diff
@@ -0,0 +1,42 @@
+// This module is included in the following assembly:
+//
+//  *openshift_pipelines/op-creating-applications-with-cicd-pipelines.adoc
+
+[id="creating-webhooks_{context}"]
+= Creating Webhooks
+
+_Webhooks_ are HTTP POST messages that are received by the EventListeners whenever a configured event occurs in your repository. The event payload is then mapped to TriggerBindings, and processed by TriggerTemplates which eventually starts a PipelineRun(s) leading to the creation and deployment of Kubernetes resources.
+
+In this section you will configure a Webhook URL on your forked Git repositories--front-end `vote-ui` and back-end `vote-api`. This URL points to the EventListener service route which you made publicly accessible in the previous step.
```

Comment:
The previous step is in the previous module. Wonder if we can simply say - This URL points to the publicly accessible EventListener service route?
Also, maybe modify the note in the previous module: ...Service must be exposed as an OpenShift Container Platform route to make it publicly accessible for configuring Webhook URLs. ?